README.md

```diff
@@ -26,6 +26,20 @@ sudo docker run \
   gcr.io/cadvisor/cadvisor:$VERSION
 ```
 
+**Note**: For a redhat docker host the above command throws oci errors. Please use the command below if the host is redhat:
```

Comment:
Can you:
- Provide information on RHEL version that is affected
- Improve letter case (RedHat, Docker etc)
- Move information to docs/running.md where some information about RHEL and Centos is already provided.

Reply:
will do, thank you, and then I will open a ticket as well - the memory dial stays at zero when the docker host is RedHat.

Reply:
I followed @bobbypage 's advice in the next comment and did a git pull from google/cadvisor and pushed the latest to adityai/advisor. How do I get the scan or test to run again?

Reply:
@iwankgb - I updated as suggested - RHEL version 7, improved letter case RedHat and Docker and I moved the command from readme.md to docs\running.md. Please review.